src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7706,4 +7706,22 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureRefReadonlyParameters" xml:space="preserve">
     <value>ref readonly parameters</value>
   </data>
+  <data name="WRN_OverridingDifferentRefness" xml:space="preserve">
+    <value>Modifier of parameter '{0}' doesn't match the corresponding parameter '{1}' in overridden or implemented member.</value>
+  </data>
+  <data name="WRN_OverridingDifferentRefness_Title" xml:space="preserve">
+    <value>Modifier of parameter doesn't match the corresponding parameter in overridden or implemented member.</value>
+  </data>
+  <data name="WRN_HidingDifferentRefness" xml:space="preserve">
+    <value>Modifier of parameter '{0}' doesn't match the corresponding parameter '{1}' in hidden member.</value>
+  </data>
+  <data name="WRN_HidingDifferentRefness_Title" xml:space="preserve">
+    <value>Modifier of parameter doesn't match the corresponding parameter in hidden member.</value>
+  </data>
+  <data name="WRN_PartialDifferentRefness" xml:space="preserve">
+    <value>Modifier of parameter '{0}' doesn't match the corresponding parameter '{1}' in partial declaration.</value>
```

Comment:
>Modifier of parameter '{0}' doesn't match the corresponding parameter '{1}' in partial declaration. [](http://example.com/codeflow?start=11&length=99)

I think mismatch in partial should be an error. First of all the spec doesn't say anything explicitly. Second, it is a mismatch in the same type. Third, if we allow the mismatch, how do we bind the body and what do we emit?